src/env.json

```diff
@@ -0,0 +1,9 @@
+{
+  "API_HOST": false,
+  "UPLOAD_METHOD": false,
+  "S3_FILES_HOST": false,
```

Comment:
Not needed

Reply:
it is actually used, should it be deleted?